src/pudl/workspace/setup_cli.py

```diff
@@ -22,22 +22,26 @@
 
 The directory structure set up for PUDL looks like this:
 
-PUDL_IN
-  └── data
-      ├── censusdp1tract
-      ├── eia860
-      ├── eia860m
-      ├── eia861
-      ├── eia923
-      ├── epacems
-      ├── ferc1
-      ├── ferc714
-      └── tmp
-
-PUDL_OUT
-  ├── parquet
-  ├── settings
-  └── sqlite
+PUDL_DIR
+  └── settings
+
+PUDL_OUTPUT
+  └── hourly_emissions_cems
+
+Initially, the directories in the data store will be empty. The pudl_datastore or
+PUDL_INPUT
+  ├── censusdp1tract
+  ├── eia860
+  ├── eia860m
+  ├── eia861
+  ├── eia923
+  ├── epacems
+  ├── ferc1
```

Comment:
I think we also already have some other directories in here, if we want to include them:
- `ferc2`
- `ferc6`
- `ferc60`
- `eia_bulk_elec`
- `epacamd_eia`

Or maybe it makes sense to generate these automatically based on whatever the currently available list of data sources in the datastore are?

Or maybe we just add some elipses in the middle and indicate that there will be one directory for every raw data source?

Reply:
Ellipses sound great to me...